criu/mem.c

```diff
@@ -191,7 +213,7 @@ static int generate_iovs(struct pstree_item *item, struct vma_area *vma, struct
 			 bool has_parent)
 {
 	unsigned long nr_scanned;
-	unsigned long pages[3] = {};
+	unsigned long pages[4] = {};
```

Comment:
AFAICS, this is used to update counters and for logging at the end of function. Should we add a counter and logging for guard page too?